addons/l10n_ca/views/report_template.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="l10n_ca_external_layout_standard" inherit_id="web.external_layout_standard">
+        <xpath expr="//div[@class='row'][last()]" position="after">
+            <div>PST: <span t-field="o.company_id.l10n_ca_pst"/></div>
+        </xpath>
+    </template>
+
+    <template id="l10n_ca_external_layout_clean" inherit_id="web.external_layout_clean">
+        <xpath expr="//div[@class='row']" position="after">
+            <div>PST: <span t-field="o.company_id.l10n_ca_pst"/></div>
+
+        </xpath>
+    </template>
+
+    <template id="l10n_ca_external_layout_boxed" inherit_id="web.external_layout_boxed">
+        <xpath expr="//div[@class='row mb8']" position="after">
+            <div>PST: <span t-field="o.company_id.l10n_ca_pst"/></div>
+        </xpath>
+    </template>
+
+    <template id="l10n_ca_external_layout_background" inherit_id="web.external_layout_background">
+        <xpath expr="//div[@class='float-left company_address']" position="after">
+            <div>PST: <span t-field="o.company_id.l10n_ca_pst"/></div>
```

Comment:
What if you are showing this in a non Canadian company?  (multi-company e.g.)

Reply:
Now I display the PST only if the company is located in Canada and the PST is present as for the partner_id.l10n_ca_pst

Thanks

Reply:
The l10n_ca_pst is actually sufficient...

Reply:
I don't know, if you say it I can do it
I found it weird because you can set a PST for any company even when it's not canadian. This is what you also do for the partner_id, why do it differently here?